src/lineup/ALineUpView2.ts

```diff
@@ -78,6 +83,7 @@ export abstract class ALineUpView2 extends AView {
 
     this.$node.classed('lineup', true);
 
+    // hack in for providing the data provider within the graph
```

Comment:
no, I was just wondering why the resolve function is saved and then resolved below. I thought alternatively the `updateRef` function could return a promise and then resolve it or something. but I don't know most of that code, so I was just curious